src/styles/episodic/marks.css

```
@@ -0,0 +1,355 @@
+
+.signal-field span {
+  width: 100%;
+  min-width: 1px;
+  /* A fixed pseudo-random envelope: the same shape on every load, because a
+     reference that reshuffles between visits is noise.
+     Every term is a property, so a variant retunes the shape rather than
+     restating the formula. The rates matter most at short lengths: the
+     default pair repeats about every thirteen bars, which is invisible across
+     seventy-two and unmissable across twenty-six. */
+  height: calc(
+    var(--signal-floor, 14%) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)